### PR TITLE
Implement feature to request reviewers by github review API

### PR DESCRIPTION
**Pull request by hyone**
Implement feature to use github review API to assign reviewers.
( And removes the feature to notice assigned reviewers by github status API because it has become redundant )

This PR works, but uses [octokit](https://github.com/octokit/octokit.rb) (ruby github API client), whose review API support is still on the stage of [pull request](https://github.com/octokit/octokit.rb/pull/860). So, it's unstable.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
 gemspec
+
+gem 'octokit', github: 'soudy/octokit.rb', branch: 'add-review-preview'
```
Comment (masa-iwasaki) [inline Gemfile]: Set this in gemspec by using `spec.add_runtime_dependency`

Comment (masa-iwasaki) [inline Gemfile]: Set this in gemspec by using `spec.add_runtime_dependency`

Comment (hyone) [inline Gemfile]: Yes it should, but `spec.add_runtime_dependency` can not specify gem via github repository.
So I have to place it in `Gemfile` until the new version of octokit  is released to rubygems, which supports review API.

**Comment by hyone** [comment]
Rebasing this branch on master.

**Comment by hyone** [comment]
Done.

**Comment by hyone** [comment]
New version of Octokit is comming!
This supports github Review API.
So, I push a commit to use `spec.add_runtime_dependency`, please review it again.

**Pull request merged**